bin/scrape_software_versions.py

```diff
@@ -18,6 +18,8 @@
     'StringTie': ['v_stringtie.txt', r"(\S+)"],
     'Preseq': ['v_preseq.txt', r"Version: (\S+)"],
     'RSeQC': ['v_rseqc.txt', r"read_duplication.py ([\d\.]+)"],
+    'dupRadar': ['v_dupRadar.txt', r"(\S+)"],
+    'edgeR: ['v_edgeR.txt', r"(\S+)"],
```

Comment:
Theres a missing `'` at the end of `edgeR` here. Thats probably why the tests are failing.